apps/prairielearn/src/ee/pages/instructorAiGenerateQuestion/instructorAiGenerateQuestion.ts

```diff
@@ -137,14 +142,25 @@ router.post(
         );
       }
 
+      const qid = genJobs[0]?.data['questionQid'];
+
+      const threads: AiGenerationPrompt[] = await queryRows(
```

Comment:
Two nits:

- No need for explicit typing here, type inference will do exactly what you want.
- Are these "generation prompts" or "threads" or "generation thread items"? Let's be consistent in our naming.

Reply:
renamed in 8f91ed5.